model/countries/NL/NL-parsing-rules.yaml

```diff
@@ -0,0 +1,143 @@
+
+  # Regular expression to match separator of "building" and "unit".
+  kHouseNumberAndUnitSeparator:
+    regex_fragment: (?:[-\s]*)
+
+capture_definitions:
+  ParseBuildingLocation:
+    capture:
+      output: building-location
+      parts:
+      - capture:
+          output: street
+          parts: [{regex_reference: kMultipleLazyWordsRe}]
+      - separator: {regex_reference: kWhitespaceSeparator}
+      - capture:
+          output: building-and-unit
+          parts:
+            - capture:
+                output: building
+                parts: [ {regex_reference: kBuildingValueRe} ]
+            - separator: {regex_reference: kHouseNumberAndUnitSeparator} 
+            - capture:
+                output: unit
+                parts: [ {regex_reference: kUnitValueRe} ]
+                quantifier: MATCH_OPTIONAL
+
+parsing_definitions:
+  building-location:
+    decomposition: 
+      capture_reference: ParseBuildingLocation
+
+  street-address-alternative-1:
+    decomposition:
+      capture: 
+        output: street-address-alternative-1
+        parts:
+        - capture_reference: ParseBuildingLocation
+
+test_parsing_definitions:
+- id: "Test 1"
+  type: building-location
+  input: "Nieuwe Hoogstraat 12-A"
+  output:
+    building-location: "Nieuwe Hoogstraat 12-A" 
+    street: "Nieuwe Hoogstraat"
+    building-and-unit: "12-A"
+    building: "12"
+    unit: "A"
+- id: "Test 2"
+  type: street-address-alternative-1
+  input: "Nieuwe Hoogstraat 12-A"
+  output:
+    street-address-alternative-1: "Nieuwe Hoogstraat 12-A"  
+    building-location: "Nieuwe Hoogstraat 12-A" 
+    street: "Nieuwe Hoogstraat"
+    building-and-unit: "12-A"
+    building: "12"
+    unit: "A"
+- id: "Test 3"
+  type: street-address-alternative-1
+  input: "Nieuwe Hoogstraat 12A"
+  output:
+    street-address-alternative-1: "Nieuwe Hoogstraat 12A"  
+    building-location: "Nieuwe Hoogstraat 12A" 
+    street: "Nieuwe Hoogstraat"
+    building-and-unit: "12A"
+    building: "12"
+    unit: "A"
+- id: "Test 4"
+  type: street-address-alternative-1
+  input: "Nieuwe Hoogstraat 12 A"
+  output:
+    street-address-alternative-1: "Nieuwe Hoogstraat 12 A"  
+    building-location: "Nieuwe Hoogstraat 12 A" 
+    street: "Nieuwe Hoogstraat"
+    building-and-unit: "12 A"
+    building: "12"
+    unit: "A"
+- id: "Test 5"
+  type: street-address-alternative-1
+  input: "Nieuwe Hoogstraat 12-bis"
+  output:
+    street-address-alternative-1: "Nieuwe Hoogstraat 12-bis"  
+    building-location: "Nieuwe Hoogstraat 12-bis" 
+    street: "Nieuwe Hoogstraat"
+    building-and-unit: "12-bis"
+    building: "12"
+    unit: "bis"
+- id: "Test 6"
+  type: street-address-alternative-1
+  input: "Nieuwe Hoogstraat 12-III"
+  output:
+    street-address-alternative-1: "Nieuwe Hoogstraat 12-III"  
+    building-location: "Nieuwe Hoogstraat 12-III" 
+    street: "Nieuwe Hoogstraat"
+    building-and-unit: "12-III"
+    building: "12"
+    unit: "III"
```

Comment:
Could you also add a test where we have `Nieuwe Hoogstraat 12II` as an input? I'd expect it should be parsed as building: 12 and unit: II